Carbon.ScheduledTasks/Carbon.ScheduledTasks.psm1

```diff
@@ -12,20 +12,166 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
+using namespace System.Collections.Generic;

```

Comment:
Is convention of using before/after #Requires and Set-StrictMode? Those almost feel 'more' important.

Reply:
PowerShell requires it be one of the first lines in the file, excluding comments.